config/config.go

```diff
@@ -777,7 +826,26 @@ func convertToTerragruntConfig(
 		terragruntConfig.IamRole = *terragruntConfigFromFile.IamRole
 	}
 
-	for _, block := range terragruntConfigFromFile.GenerateBlocks {
+	generateBlocks := []terragruntGenerateBlock{}
+	generateBlocks = append(generateBlocks, terragruntConfigFromFile.GenerateBlocks...)
+
+	if terragruntConfigFromFile.GenerateAttrs != nil {
+		remoteStateMap, err := parseCtyValueToMap(*terragruntConfigFromFile.GenerateAttrs)
```

Comment:
```suggestion
		generateMap, err := parseCtyValueToMap(*terragruntConfigFromFile.GenerateAttrs)
```

Reply:
Whoops, good catch. Fixed in https://github.com/gruntwork-io/terragrunt/pull/1639/commits/b6f61c8300610841e5f4ae4b466f5d76490c0374.